packages/odyssey-storybook/src/components/odyssey-mui/Toast/Toast.stories.tsx

```diff
@@ -136,62 +100,62 @@ InfoStatic.args = {};
 
 export const Error = DefaultTemplate.bind({});
 Error.args = {
-  content: "Security breach in Hangar 18",
+  text: "Security breach in Hangar 18",
   role: "alert",
   severity: "error",
 };
 
 export const ErrorStatic = StaticTemplate.bind({});
 ErrorStatic.args = {
-  content: "Security breach in Hangar 18",
+  text: "Security breach in Hangar 18",
   role: "alert",
   severity: "error",
 };
 
 export const Warning = DefaultTemplate.bind({});
 Warning.args = {
-  content: "Severe solar winds may delay local system flights",
+  text: "Severe solar winds may delay local system flights",
   role: "status",
   severity: "warning",
 };
 
 export const WarningStatic = StaticTemplate.bind({});
 WarningStatic.args = {
-  content: "Severe solar winds may delay local system flights",
+  text: "Severe solar winds may delay local system flights",
   role: "status",
   severity: "warning",
 };
 
 export const Success = DefaultTemplate.bind({});
 Success.args = {
-  content: "Docking completed",
+  text: "Docking completed",
   role: "status",
   severity: "success",
 };
 
 export const SuccessStatic = StaticTemplate.bind({});
 SuccessStatic.args = {
-  content: "Docking completed",
+  text: "Docking completed",
   role: "status",
   severity: "success",
 };
 
 export const Dismissible = DefaultTemplate.bind({});
 Dismissible.args = {
-  actionLink: (
-    <Link href="#anchor" variant="monochrome">
-      View report
-    </Link>
-  ),
   isDismissible: true,
+  linkText: "View report",
+  linkUrl: "#",
+  onClose: () => {
+    return true;
+  },
 };
 
 export const DismissibleStatic = StaticTemplate.bind({});
 DismissibleStatic.args = {
-  actionLink: (
-    <Link href="#anchor" variant="monochrome">
-      View report
-    </Link>
-  ),
   isDismissible: true,
```

Comment:
You can cut these from the stories since they don't apply anymore.

Reply:
Just to confirm, I think I already did — is there something else I should do?